tests/js/list-tests.js

```diff
@@ -1412,13 +1412,72 @@ module.exports = {
 
     testCreateEmbeddedObjects: function() {
         const realm = new Realm({schema: [schemas.ContactSchema, schemas.AddressSchema]});
+        realm.write(() => { // FIXME: This should not be required!
```

Comment:
is this FIXME for now ?

Reply:
Yes, it is fixed on the `kneth/stitch-integration` branch (by fixing `Realm.clearTestState()`).